flink-table/flink-table-planner-blink/pom.xml

```diff
@@ -378,6 +378,8 @@ under the License.
 
 									<!-- flink-table-runtime-blink dependencies -->
 									<include>org.codehaus.janino:*</include>
+									<include>org.apache.flink:flink-table-code-splitter</include>
```

Comment:
Can include in runtime?

Reply:
Runtime module currently does not have its own packaging process. As Janino dependency is also packed here, I think we should add the dependency here.